src/server/wtr-config.js

```diff
@@ -231,7 +231,7 @@ export class WTRConfig {
 		}
 
 		// convert all browsers to playwright
-		config.groups.forEach(g => g.browsers = this.getBrowsers(g.browsers));
+		config.groups.forEach(g => g.browsers = this.getBrowsers(g.browsers, config.deviceScaleFactor ?? (g.name === 'vdiff' ? 2 : 1)));
```

Comment:
If we don't want to mess with switching this back to 1 for everything non-`vdiff`, we could just do:
```
config.deviceScaleFactor ?? 2
```
or
```
g.name === 'vdiff' && config.deviceScaleFactor ? config.deviceScaleFactor : 2
```